app/steps/ui/startapply/template.html

```diff
@@ -0,0 +1,35 @@
+{% extends "includes/layout.html" %}
+{% block page_title %}
+{{ content.title }}
+{% endblock %}
+
+{% block content %}
+
+{% from "widgets/lists.html" import taskAction, unorderedList %}
+
+<main id="content" role="main">
+    {% include "includes/phase_banner_beta.html" %}
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <h1 class="form-title heading-large">{{content.header}}</h1>
+            <p>{{content.introduction | safe}}</p>
+            <p>{{content.paragraph1 | safe}}</p>
+            <div class="heading-medium">
+                {{content.feesHeader}}
+            </div>
```

Comment:
It wold be more semantically correct for this heading to be `<h2>` rather than `<div>`